src/transformers/models/beit/modeling_beit.py

```diff
@@ -361,6 +361,52 @@ def forward(
         return outputs
 
 
+class BeitSdpaSelfAttention(BeitSelfAttention):
```

Comment:
Did you copy it from some model entirely or it have been modified? I mean it would be nice to add "#Copied from" if we could

Reply:
Actually this particular attention was a bit different because of the relative bias. So I modified it accordingly. The SelfAttention itself was not copied from anywhere.